packages/web-ui/app/rooms/[room]/layout.tsx

```diff
@@ -35,6 +41,11 @@ export default async function RoomLayout({
   params: { room: string };
 }) {
   const room = await getRoom(params.room);
+
+  if (!room) {
+    redirect('/');
+  }
+
```

Comment:
Can't this be done directly in line 27?

Reply:
I tried, but the `redirect` fails and the page is rendered anyway

<img width="867" alt="image" src="https://github.com/xgeekshq/intelli-mate/assets/97113028/5267d16f-e07f-416b-a6a7-bf0df01906fb">